spec/requests/reset_password_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+describe 'ResetPassword', type: :request do
+  let(:user) { create(:user) }
+
+  it 'get forgot password form' do
+    get '/forgot_password'
+    expect(response).to have_http_status(:success)
+  end
+
+  it 'get reset password form' do
+    get '/reset_password'
+    expect(response).to have_http_status(:success)
+  end
+
+  context 'right params when send letter' do
+    it 'return deliveries.size = 1' do
+      post '/auth/password', params: { email: user.email, redirect_url: 'http://localhost:3000/reset_password' }
+      expect(ActionMailer::Base.deliveries.size).to eq(1)
```

Comment:
https://relishapp.com/rspec/rspec-expectations/v/3-8/docs/built-in-matchers/change-matcher